todo/settings.py

```diff
@@ -39,9 +39,11 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'tasks.apps.TasksConfig',
+    #'tasks.apps.TasksConfig',
```

Comment:
I originally commented this out cause I had made changes to files within tasks folder and wanted access to the entire folder instead of specifically `tasks.apps.TasksConfig` but since we're reverting back, I'll add this back in.